sprint_1/src/laura/uebungsprojekt/aufgaben/UPLK79_EinerVonVielen.java

```diff
@@ -0,0 +1,27 @@
+package laura.uebungsprojekt.aufgaben;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class UPLK79_EinerVonVielen {
+
+    private static List<Character> convertStringToCharList(String str) {
+        List<Character> chars = new ArrayList<>();
+        for (char ch : str.toCharArray()) {
+            chars.add(ch);
+        }
+        return chars;
+    }
+
+    public static void findXinWord() {
+        Scanner sc = new Scanner(System.in);
+        System.out.print("Enter a word and a number: ");
+        String word = sc.nextLine();
+        int number = sc.nextInt();
+        List<Character> charList = convertStringToCharList(word);
```

Comment:
Die Klasse `String` hat in Java eine `.charAt(index)`-Methode - damit hättest du dir das Umwandeln in eine `char`-Liste sparen können ;)